src/entities/twin/libs/helpers.ts

```diff
@@ -42,6 +62,17 @@ export const buildFilters = (filters: FiltersState): Record<string, any> => {
   );
 };
 
+const getKeyByFilter = (filterKey: string): string => {
```

Comment:
This function introduces unnecessary complexity. Let's remove it and simplify the logic for `buildFilters`.

Please rewrite `buildFilters` to match the streamlined approach used [here](https://github.com/alcosi/twinfaces/blob/2908aaaaadd19246f40dd089c761072faaf1aec8/src/entities/permission/libs/helpers.ts#L10).

Thanks!